CFC_DataCollector/common/utils.py

```diff
@@ -0,0 +1,17 @@
+#return True if only one mode of travel present
+#usually one moves activity..
+def get_single_mode(json_section):
+    return
+
+#provide from user
+def get_cost(): return
```

Comment:
Should functions like get_cost() be part of the trip, or part of the model building? After all, it is not clear if we care about cost unless we want to use it as part of a model...